scanomatic/data_processing/pheno/state.py

```diff
@@ -35,25 +35,14 @@ class PhenotyperSettings:
     gaussian_filter_sigma: float
     linear_regression_size: int
     phenotypes_inclusion: Optional[PhenotypeDataType]
-    no_growth_monotonicity_threshold: float
-    no_growth_pop_doublings_threshold: float
+    no_growth_monotonicity_threshold: Optional[float]
+    no_growth_pop_doublings_threshold: Optional[float]
```

Comment:
these should have default `None` right? and `phenotypes_inclusion` shouldn't really be optional but have a default value too right?

Reply:
`phenotypes_inclusion` was strictly optional in the saved settings, but it should default to `PhenotypeDataType.Trusted`.

Reply:
so better `no_growth_pop_doublings_threshold: Optional[float] = None` etc